app/components/hero_component.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class HeroComponent < ViewComponent::Base
+  def initialize(title, subtitle = '', schema = nil)
+    super
+    @title = clean_title(title)
+    @subtitle = subtitle
+    @schema = schema
+  end
+
+  def clean_title(title)
+    return title.length > 32 ? title.split.in_groups(2, false).map { |g| g.join(' ') }.join('<br> ').html_safe : title # rubocop:disable Rails/OutputSafety
```

Comment:
return is implicit in ruby - the last line of any function auto returns.

perhaps this ternary should be converted into an if so we can remove that rubocop:disable statement?

Reply:
the rubocop is unhappy with the labelling of it as html_safe. I could probably rewrite this to generate the html code rather than just copy the existing code though.

Reply:
ohh i think this might be one of those "lots of hours have been wasted here" places lol, might be best to just leave it